tests/30rooms/10redactions.pl

```diff
@@ -28,16 +28,22 @@ =head2 matrix_redact_event
 
 =cut
 
+sub random_transaction_id
+{
+   join "", map { chr 65 + rand 26 } 1 .. 20;
```

Comment:
```suggestion
   join "", map { chr( 65 + rand 26 )} 1 .. 20;
```

Reply:
I copied the code from: https://github.com/matrix-org/sytest/blob/fbc57e787703e7fcdbd3ab1b622a773c7bc7cc19/tests/52user-directory/01public.pl#L456

Should I change it there as well? There are a few more occurrences in the codebase with the same pattern.

Reply:
I don't know if it's necessary to go and change existing code. But I think it's better to strive for clarity rather than cargo-cult existing unclear code.